<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>空号检测器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>0</y>
      <width>256</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>10</y>
      <width>211</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string> 在下方输入号码 </string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>20</y>
       <width>191</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_join">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>加入左方列表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_check">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>三五检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_export">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>导 出 数 据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_check_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>350</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>航海检测</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>激活码</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
